### PR TITLE
New Version: Sigil-Ebook.Sigil version 2.1.0

### PR DESCRIPTION
**Pull request by dougmassay**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/147284)

### CODE REVIEW

**Comment by dougmassay** [comment]
I need just a little more to go on than "issue with installing the application" nothing has changed regarding the installer. It works fine everywhere I've tried it.

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-04-01 18:03:47.191 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!00007FFA4CC1C11F: (caller: 00007FFA4C9A8D6F) LogHr(1) tid(1cf8) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFA4CB22B08: (caller: 00007FFA4CBCBC20) Exception(1) tid(1cf8) 8A15005E ] 
> 2024-04-01 18:03:47.191 [REPO] Failed to open available source: msstore
> 2024-04-01 18:03:54.851 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!00007FFA31D3C11F: (caller: 00007FFA31AC8D6F) LogHr(1) tid(67c) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFA31C42B08: (caller: 00007FFA31CEBC20) Exception(1) tid(67c) 8A15005E ] 
> 2024-04-01 18:03:54.851 [REPO] Failed to open available source: msstore
> 2024-04-01 18:03:56.122 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.38.33135.0

(Automated response - build 876.)

**Comment by dougmassay** [comment]
So it can't install the latest Microsoft.VCRedist.2015+.x64 dependency via it's winget manifest? I'm unaware of anything I can do to affect how that particular dependency gets installed. I've named the dependency correctly in my installer yaml. Surely I don't need to specify an exact version of the VCRedist.2015+.x64 package?

**Comment by dougmassay** [comment]
Please advise. I'm seeing this error all over the place here. I can't fix a broken VCRedist.2015+.x64 dependency install by pushing more changes to my branch.

**Comment by dougmassay** [comment]
#147337

**Comment by dougmassay** [comment]
As mentioned previously, there is nothing for me to "fix." something has changed RE dependency packages that is breaking the Azure Pipeline.

See https://github.com/microsoft/winget-pkgs/issues/147337

No ability to see any detailed logs referenced above - https://github.com/microsoft/winget-pkgs/pull/147284#issuecomment-2030957187 - concerning the error.

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-04-02 05:05:18.989 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!00007FFEE493C11F: (caller: 00007FFEE46C8D6F) LogHr(1) tid(1780) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFEE4842B08: (caller: 00007FFEE48EBC20) Exception(1) tid(1780) 8A15005E ] 
> 2024-04-02 05:05:18.989 [REPO] Failed to open available source: msstore
> 2024-04-02 05:05:26.148 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!00007FFECC74C11F: (caller: 00007FFECC4D8D6F) LogHr(1) tid(18d4) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFECC652B08: (caller: 00007FFECC6FBC20) Exception(1) tid(18d4) 8A15005E ] 
> 2024-04-02 05:05:26.148 [REPO] Failed to open available source: msstore
> 2024-04-02 05:05:27.423 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.38.33135.0

(Automated response - build 876.)

**Comment by dougmassay** [comment]
> The issue seems to be a scope mismatch. I believe the dependency only has machine scope, which is conflicting with the user scope here.

There was never a scope mismatch in the past. My yaml hasn't really changed (other than urls, shas, and version numbers) in a long time. If the dependency has changed their scope options, I'm at a loss as to how my package can possibly be installed with the user scope anymore. 

I'd rather not have to assume the necessary vc++ redist libraries are already installed by removing the dependency from my installer yaml altogether.

**Comment by dougmassay** [comment]
It could still be a scope issue, but if it is, its not because of any change to the Microsoft.VCRedist.2015+.x64 package itself. It's never had user scope (and no history of it it ever being removed), and my manifest never failed because of it in the past. There has to have been some sort of change in how the InstallerVerification pipeline handles PackageDependencies and multiple scopes.

**Comment by dougmassay** [comment]
It's not at all obvious what sort of feedback is currently being requested of me.

File: manifests/s/Sigil-Ebook/Sigil/2.1.0/Sigil-Ebook.Sigil.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  Scope: machine
+  Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+  InstallerUrl: https://github.com/Sigil-Ebook/Sigil/releases/download/2.1.0/Sigil-2.1.0-Windows-x64-Setup.exe
+  InstallerSha256: 1048E93AFBB64D7DBFA61BE80CFB677F1C6B6BF72EFD4B89C891A358E34094BD
+  InstallerSwitches:
+    Custom: /ALLUSERS /NORESTART /SUPPRESSMSGBOXES /COMPONENTS="dicon,*afiles"
+ManifestType: installer
+ManifestVersion: 1.1.0
```
Comment (stephengillie) [inline manifests/s/Sigil-Ebook/Sigil/2.1.0/Sigil-Ebook.Sigil.installer.yaml:36]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/s/Sigil-Ebook/Sigil/2.1.0/Sigil-Ebook.Sigil.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Sigil-Ebook.Sigil
+PackageVersion: 2.1.0
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.1.0
```
Comment (stephengillie) [inline manifests/s/Sigil-Ebook/Sigil/2.1.0/Sigil-Ebook.Sigil.yaml:8]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/s/Sigil-Ebook/Sigil/2.1.0/Sigil-Ebook.Sigil.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ShortDescription: Sigil is a free, open source, multi-platform ebook editor that uses Qt (and QtWebEngine). It is designed to edit books in ePub format (both ePub 2 and ePub 3).
+Moniker: sigil
+Tags:
+- e-book
+- ebook
+- ebook-editor
+- epub
+ReleaseNotesUrl: https://github.com/Sigil-Ebook/Sigil/releases/tag/2.1.0
+ManifestType: defaultLocale
+ManifestVersion: 1.1.0
```
Comment (stephengillie) [inline manifests/s/Sigil-Ebook/Sigil/2.1.0/Sigil-Ebook.Sigil.locale.en-US.yaml:25]:
```suggestion
ManifestVersion: 1.6.0
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Sorry for not catching this earlier. We've depreciated this manifest version. But most fields are identical to the most recent version. Increasing the manifest version number is usually enough to pass this check.

**Comment by dougmassay** [comment]
It only progressed this far because I moved the Dependencies property from the manifest root to the machine scope installer node to see if a scope mismatch was the culprit. Seems it was. But doing so also made the Manifest-Metadata-Consistency tag appear. I'm not entirely happy with the Dependencies change, but I guess I can live with user scope installs having to manually make sure the dependency is is met before installing. I'll push the requested manifest version changes shortly.

File: manifests/s/Sigil-Ebook/Sigil/2.1.0/Sigil-Ebook.Sigil.installer.yaml (hunk 1 above)
Comment (dougmassay) [reply manifests/s/Sigil-Ebook/Sigil/2.1.0/Sigil-Ebook.Sigil.installer.yaml]: Change made

**Review by stephengillie — APPROVED** [review]

**Pull request merged**